bot/seasons/valentines/be_my_valentine.py

```diff
@@ -0,0 +1,237 @@
+import logging
+import random
+import typing
+from json import load
+from pathlib import Path
+
+
+import discord
+from discord.ext import commands
+from discord.ext.commands.cooldowns import BucketType
+
+from bot.constants import Colours, Lovefest
+
+log = logging.getLogger(__name__)
+
+HEART_EMOJIS = [":heart:", ":gift_heart:", ":revolving_hearts:", ":sparkling_heart:", ":two_hearts:"]
+EMOJI_1 = random.choice(HEART_EMOJIS)
+EMOJI_2 = random.choice(HEART_EMOJIS)
+USER_LOVEFEST = []
+
+
+class BeMyValentine:
+    """
+    A cog that sends valentines to other users !
+    """
+    id = Lovefest.role_id
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="lovefest")
+    async def add_role(self, ctx):
+        """
+            This command adds people to the lovefest role
```

Comment:
This should be the same indent as the quotation marks.
It should probably also end in a period.